perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AttributesManagerMethod.java

```diff
@@ -2517,7 +2655,15 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 			}
 
 			if (parms.contains("facility")) {
-				if (parms.contains("resource") && parms.contains("user") && parms.contains("member")) {
+				if (parms.contains("resource") && parms.contains("group") && parms.contains("user") && parms.contains("member")) {
```

Comment:
Same like in the last comment. Just think about it. Of course this way technically works too.